etna/search/tests/test_forms.py

```diff
@@ -4,31 +4,79 @@
 
 
 class CatalogueSearchFormTest(SimpleTestCase):
-    def test_start_date_after_end_date_is_invalid(self):
+    def test_various_opening_date_inputs_is_valid(self):
+        """tests inputs accross both date fields with various input formats for form validity"""
+        for label, value in (
+            (
+                "input end date with empty start date",
+                {
+                    "input": {
```

Comment:
Not sure what we're gaining from the multi-level dicts here. If all we need is data for the form, we should lose the outer dicts (with the `"input"` key)

Reply:
Done - removed input key